README.md

```diff
@@ -76,7 +76,7 @@ Current available templates include:
 
 - [netlify] - Netlify CMS template using preact.
 
-- [typescript] - Default template implemented in TypeScript
+- [typescript] - Default template implemented in TypeScript. NOTE: TypeScript by default does not support `async!` import syntax.
```

Comment:
I don't think we need a note attached to the template itself. `async!` support probably isn't the main selling point for the template. The fact that it uses TS is.